android/res/values-ru/strings.xml

```diff
@@ -550,7 +550,7 @@
 	<!-- Cuisine -->
 	<string name="cuisine_chinese">китайская кухня</string>
 	<!-- Cuisine -->
-	<string name="cuisine_coffee_shop">кафе</string>
+	<string name="cuisine_coffee_shop">Кофе</string>
```

Comment:
Проверь сам в словаре.

> On 18.3.2016, at 15.36, Ilya Zverev <notifications@github.com> wrote:
> 
> In android/res/values-ru/strings.xml:
> 
> > @@ -550,7 +550,7 @@
> >  	<!-- Cuisine -->
> >  	<string name="cuisine_chinese">китайская кухня</string>
> >  	<!-- Cuisine -->
> > -	<string name="cuisine_coffee_shop">кафе</string>
> > +	<string name="cuisine_coffee_shop">Кофе</string>
> О, а тут кофе, хотя на других языках — кафе.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>